pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -698,32 +698,51 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podContainerC
 		result.Fail(fmt.Errorf("unable to get resource configuration processing resize for pod %s", pod.Name))
 		return
 	}
+	currentPodMemoryConfig, err := pcm.GetPodCgroupConfig(pod, v1.ResourceMemory)
+	if err != nil {
+		klog.ErrorS(nil, "Unable to get pod cgroup memory config", "pod", pod.Name)
```

Comment:
nit: use `klog.KObj(pod)` for the pod reference. Same below, and you can fix the log on line 697 above while you're at it.
```suggestion
		klog.ErrorS(nil, "Unable to get pod cgroup memory config", "pod", klog.KObj(pod))
```